source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5468,8 +5462,12 @@ export class Container extends StylableCardElementContainer {
 
         let element = document.createElement("div");
 
-        if (this.rtl !== undefined && this.rtl) {
+        if (this.rtl === undefined) {
+            element.removeAttribute("dir");
```

Comment:
Explicitly removing the attribute shouldn't be necessary, the attribute isn't set by default on newly created elements.

Reply:
Fixed